src/frontend/src/views/AgreeTerms/index.tsx

```diff
@@ -11,7 +11,7 @@ import ENV from "src/common/constants/ENV";
 import { useUpdateUserInfo } from "src/common/queries/auth";
 import { ROUTES } from "src/common/routes";
 import { PageContent } from "../../common/styles/mixins/global";
-import { Details, Title } from "./style";
+import { BoldText, Details, Title } from "./style";
```

Comment:
Looking at the `Privacy` and `Terms` component pages, they're importing from a common style for some of these. (see https://github.com/chanzuckerberg/aspen/blob/9572eccaf89903c36e5ec7acfa6eabe1ba789c78/src/frontend/src/views/Terms/index.tsx#L15 )

I'm not sure if we should be trying to standardize to that and use the same here as well. It might be sensible to have it break off -- your `BoldText` is slightly different, for example -- but I just don't know which direction we're aiming.